src/Tribe/Attendee_Registration/Main.php

```diff
@@ -211,18 +211,17 @@ public function get_attendee_registration_page() {
 	 * Check if the modal is enabled.
 	 *
 	 * @since 4.11.0
+	 * @since TBD Defaulted ticket-attendee-modal setting to true if never set.
 	 *
 	 * @param int|WP_Post|null $post The post (or its ID) we're testing. Defaults to null.
 	 *
 	 * @return boolean
 	 */
 	public function is_modal_enabled( $post = null ) {
-		/** @var $settings_manager Tribe__Settings_Manager */
-		$settings_manager = tribe( 'settings.manager' );
 
-		$event_tickets_plus = class_exists( 'Tribe__Tickets_Plus__Main' );
+		$event_tickets_plus = class_exists( 'Tribe__Tickets_Plus__Main', false );
 
-		$option = $settings_manager::get_option( 'ticket-attendee-modal' );
+		$option = tribe_get_option( 'ticket-attendee-modal', true );
```

Comment:
`Tribe__Settings_Manager::get_option()` has a parameter for a default value. Why did you choose to go with `tribe_get_option()`?

Reply:
I was originally looking at how some of our other options are grabbed and they were using `tribe_get_option`. It seems like `tribe_get_option` actually calls `Tribe__Settings_Manager` with a filter as well.